lib/rules/web/admin_console/4.3/actions.xyaml

```diff
@@ -1,15 +1,20 @@
 click_one_dropdown_action:
   action: click_resource_action_button
-  action: choose_item_from_list
+  action: choose_action_item_from_list
 click_resource_action_button:
   elements:
   - selector:
       xpath: //button[contains(.,'Actions')]
     op: click
+choose_action_item_from_list:
```

Comment:
Use different rules for action downdrop item and common list item since they have different attributes.